bitwatch/umbrel-app.yml

```diff
@@ -0,0 +1,32 @@
+manifestVersion: 1
+id: bitwatch
+category: bitcoin
+name: Bitwatch
+version: "1.4.4"
+tagline: Monitor Bitcoin addresses in real-time
+description: >-
+  Monitor Bitcoin addresses in the mempool and on-chain using the mempool.space API and websocket.
+
+  Track balance changes, get telegram notifications, and manage expectations for multiple addresses.
+
+  Features:
```

Comment:
Looks good @zapomatic, just make sure to use 2 empty lines to add some space between the paragraphs:
![CleanShot 2025-04-07 at 16 57 29](https://github.com/user-attachments/assets/73e7d87b-cc0b-4436-9231-6b9bb857eaaf)

```suggestion
  Monitor Bitcoin addresses in the mempool and on-chain using the mempool.space API and websocket.


  Track balance changes, get telegram notifications, and manage expectations for multiple addresses.


  Features:
```